aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2152,7 +2152,8 @@ Tensor flatten(const Tensor& self, DimnameList dims, Dimname out_dim) {
 }
 
 Tensor ravel(const Tensor& self) {
-  return self.reshape(-1);
+  // If self is contiguous, `contiguous()` is a no-op
```

Comment:
This is a true statement, but why is this comment here? I would probably remove it or add some context to it so readers understand why it's here